### PR TITLE
Replace Alef character with SVG

### LINKED ISSUE #176: Replace the "ALEF" utf8 symbol with a SVG
For consistency sake across OSes. As of today, the symbol is rendered in very different ways.

Note: this component could later be shared in a "common" UI library.

### PR DESCRIPTION
**Pull request by nop33**
Closes #176

### CODE REVIEW

File: src/components/AlefSymbol.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import alefSymbol from '../images/alef.svg'
+
+interface AlefSymbolProps {
+  color?: string
+  className?: string
+}
+
+const AlefSymbol = ({ className, color }: AlefSymbolProps) => {
+  return (
```
Comment (lf94) [inline src/components/AlefSymbol.tsx:29]: Maybe to save you and future us some pain we can use `useTheme` here to get the color? Then the symbol can color itself.

**Review by lf94 — COMMENTED** [review]
Other than my one comment your code is great :+1: 

I still need to do a physical/visual test.

File: src/components/AlefSymbol.tsx (hunk 1 above)
Comment (nop33) [reply src/components/AlefSymbol.tsx:29]: I don't see how using this hook can help me customize the color of the symbol based on the color of the text where it is used at. I need to set it to red when used in error messages, gray when used in input field labels, and green when used in confirmed transaction details.

File: src/components/Inputs/index.tsx [hunk 2]
```diff
@@ -20,11 +20,12 @@ import { motion, Variants } from 'framer-motion'
 import styled, { css } from 'styled-components'
 import tinycolor from 'tinycolor2'
 
-export interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
-  error?: string
+export interface InputProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'placeholder'> {
```
Comment (lf94) [inline src/components/Inputs/index.tsx:23]: Mmm, `placeholder` not already optional on `HTMLInputElement`? Interesting they did that.

Comment (nop33) [reply src/components/Inputs/index.tsx:23]: I need to override it because it expects `string | undefined` and it conflicts with the `null` or `ReactNode`.

**Comment by lf94** [comment]
@nop33 I'm quite busy actually and I trust what you've done, I'll approve and merge.

**Review by lf94 — APPROVED** [review]

**Comment by mvaivre** [comment]
Mhh, I was actually reviewing it, and testing "visually".
When we're not in a hurry, let's wait for all reviews before merging. Or at least ask for confirmation before merging. 
Thanks!

**Pull request merged**